src/app/components/result-box/result-box.component.ts

```diff
@@ -1,17 +1,20 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
 @Component({
   selector: 'app-result-box',
   templateUrl: './result-box.component.html',
   styleUrls: ['./result-box.component.scss'],
 })
 export class ResultBoxComponent {
-  @Input() word = '';
-  @Input() borderResultStyles = false;
+  @Input() word = ''
+  @Input() splitWord: String[] = [];
```

Comment:
```suggestion
  @Input() splitWord: string[] = [];
```

https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string

Reply:
change all ocurrences